webcomponents/tool/src/main/frontend/js/conversations/sakai-conversations.js

```diff
@@ -299,6 +299,8 @@ export class SakaiConversations extends SakaiElement {
     } else {
       this.state = STATE_NOTHING_SELECTED;
     }
+
+    location.reload();
```

Comment:
Can we use `this.requestUpdate();` ?

Reply:
No, because the permissions data is stale. We could fetch the data again and that would trigger updates, but it's a lot clearer just reloading. It doesn't happen very often, either.